### DISCUSSION TITLE
Decide what to do about the SourceForge and danse source code archives and the archive repo

### DISCUSSION THREAD

**Discussion by smk78** [SasView/sasview#2344] [General]
_This issue has been written as an aide memoir for @butlerpd and @smk78!_

A long-standing issue has been to sunset the old SANSView/SasView SourceForge site. A pre-requisite for this is ensuring important content is securely archived elsewhere, and GitHub would be the logical place.

A similar, but arguably less urgent, argument applies to legacy content on the danse server.

The SasView\archive repo was created for this purpose, and is functional. However, a problem has arisen.

It transpires that GitHub will not track files >~50Mb, or >~25Mb if using the browser interface. For such files, GitHub recommends doing one of two things:

- either, adding these files using [Git Large File Support (Git-LFS)](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-git-large-file-storage),
- or, by adding these files as releases.

For simplicity, @smk78 migrated the legacy .gz, .tgz, and .zip distribution archives to \archive using Git-LFS.

This has worked well, except the volume of data has slightly exceeded (1.06Gb) the free 1Gb LFS limit that GitHub makes available per organization.

There is no immediate threat to the content in \archive (see [here](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-storage-and-bandwidth-usage)) and the repo can still be cloned, but no additional content _anywhere_ in the SasView organization can now be added by LFS **(adding by normal commits is unaffected!)**.

To add further content by LFS would require removing existing content, or purchasing a monthly Git-LFS 'data pack' (at the time of writing $5 gets 50Gb).

Alternatively, we need to figure out another way to archive our legacy content.

**Comment by krzywon** [comment]
Based on a discussion during the technical meeting, old executables from sourceforge probably won't work on modern systems, so archiving them isn't likely useful anymore. Should we close the sourceforge and not worry about backups?
